server/core/runtime/import_step_runner_test.go

```diff
@@ -29,33 +28,57 @@ func TestImportStepRunner_Run_Success(t *testing.T) {
 		Log:                logger,
 		EscapedCommentArgs: []string{"-var", "foo=bar", "addr", "id"},
 		Workspace:          workspace,
-		RepoRelDir:         ".",
-		User:               models.User{Username: "username"},
-		Pull: models.PullRequest{
-			Num: 2,
-		},
-		BaseRepo: models.Repo{
-			FullName: "owner/repo",
-			Owner:    "owner",
-			Name:     "repo",
-		},
```

Comment:
How come this is removed from this existing test?

Reply:
These fields arevnot used in this test. I just keep only fields which are used in the test to maintain test easily.